data/110/875/806/5/1108758065.geojson

```diff
@@ -26,6 +26,9 @@
     "mz:hierarchy_label":1,
     "mz:is_current":1,
     "mz:min_zoom":6.0,
+    "name:fra_x_preferred":[
+        "Fereej Al Amir/Muraikh/Al Soudan North"
```

Comment:
Nord?

Reply:
Hmm.. second time seeing this. I'm going to merge this for now, but I'll try to pull out what look to be un-translated strings for additional review.